lib/play.rb

```diff
@@ -31,3 +31,9 @@ def turn(board)
 end
 
 # Define your play method below
+def play(board)
+  for i in 1..9
```

Comment:
Rubyists generally shy away from `for` loops and prefer constructs like:

`(1..9).each do |number|`

or:

`1.upto(9).do I number| `